src/components/MapView/Analyser/index.tsx

```diff
@@ -98,12 +103,15 @@ function Analyser({ classes }: AnalyserProps) {
           (LayerDefinitions[hazardLayerId] as WMSLayerProps)?.serverLayerName
         ]
       : null;
+    // if (Number(new URLSearchParams(history.location.search).get('date'))) {
+    //   return;
+    // }
     if (!dates || dates.length === 0) {
       setSelectedDate(null);
     } else {
       setSelectedDate(dates[dates.length - 1]);
     }
-  }, [availableDates, hazardLayerId]);
+  }, [availableDates, hazardLayerId, history.location.search]);
```

Comment:
Where did this extra dependency come from?

Reply:
An extra dependency I added for dates but forgot to remove it. Nvm.